pkg/cloud/vsphere/provisioner/govmomi/utils.go

```diff
@@ -25,6 +25,10 @@ var (
 	DefaultSSHPublicKeyFile = "/root/.ssh/vsphere_tmp.pub"
 )
 
+const (
```

Comment:
Nit: This could go into `cloud/vsphere/constants/constant.go` as well

Reply:
looks like that is the proper place, done, thx